README.md

```diff
@@ -257,4 +285,14 @@ All activities under source{d} projects are governed by the
 
 ## License
 
-GPL v3.0, see [LICENSE](LICENSE.md).
+GPL v3.0, see [LICENSE](./LICENSE.md).
+
+
+
+
+
+
+
```

Comment:
hmmmm

Reply:
hmmmm indeed :D

Reply:
Oh! come on! don't you like open space?
![gZEBpuOkPuydi](https://user-images.githubusercontent.com/2437584/59929830-ebb7ff00-9441-11e9-8256-9598f9b5a90d.gif)
ok... well... removed the extra space...